modules/core/src/main/java/org/apache/ignite/IgniteCluster.java

```diff
@@ -453,6 +453,21 @@
      */
     public void active(boolean active);
 
+    /**
+     * Checks Ignite grid in read-only mode or not.
+     *
+     * @return {@code true} if grid is in read-only mode and {@code false} If grid allows data modification operations.
```

Comment:
Here and below check for letters case (must be @code True) and same issues as above - quality of documentation.

Reply:
done